source/Introduction.rst

```diff
@@ -1,11 +1,16 @@
-.. _InstallationGuide:
-
 Introduction
 ============
 
-TODO:
+Space ROS is an open-source spacecraft flight software framework.
```

Comment:
```suggestion
Space ROS is an open-source spacecraft flight software framework.
```

Is this accurate or is it a space robotics framework that is usable (eventually, theoretically) for spacecraft flight software?

I'm a bit worried about the use of present tense without a qualifier on the maturity level of the project thus far as I wouldn't want to give people an impression of vaporware.

Reply:
Yes, I agree. I grabbed some text from another document, but will tone it down.

Reply:
I've reworked the first paragraph to mention that Space ROS is based on ROS 2 and *will be hardened* (future)